src/NuGet.Core/NuGet.Protocol/Plugins/PluginDiscoverer.cs

```diff
@@ -146,7 +146,11 @@ private IEnumerable<string> GetPluginFilePaths()
         {
             if (string.IsNullOrEmpty(_rawPluginPaths))
             {
+#if IS_DESKTOP
```

Comment:
This can be simplified - 

```
var directories = new List<string> { PluginDiscoveryUtility.GetNuGetHomePluginsPath() };

#if IS_DESKTOP
// comment explaining the extra add for desktop
directories.Add(PluginDiscoveryUtility.GetInternalPlugins());
#endif
```

Reply:
yeah good point.